packages/mobile/src/identity/securityCode.ts

```diff
@@ -78,7 +90,7 @@ async function requestValidator(
       securityCode,
     }
 
-    return attestationsWrapper.getAttestationForSecurityCode(
+    return await attestationsWrapper.getAttestationForSecurityCode(
```

Comment:
This was the original problem causing the actual error log not to be logged.
See https://celo-org.slack.com/archives/CL7BVQPHB/p1621254448334500